tools/run_docker.sh

```diff
@@ -88,7 +88,7 @@ if [[ -z "${COMMAND}" ]]; then
     exit 1
 fi
 
-DOCKER_CMD="ln -sf ${PYTHON_LIB} /usr/bin/python && ${COMMAND}"
+DOCKER_CMD="tools/ci_testing/install_py36.sh && ln -sf ${PYTHON_LIB} /usr/bin/python && ${COMMAND}"
```

Comment:
How about moving the installation to 79 line ?

```bash
case ${PYTHON} in
    py2) xxx
    py3) install_py36.sh and set environment variable
```

Reply:
I misunderstand it.

Reply:
Hmmm I believe that  would install py36 on host? We need it to run as a command in the docker container

Reply:
Oh CMD is  `ln -sf ${PYTHON_LIB}` 
so if PYTHON_LIB = "/usr/bin/python3 && install_py36.sh" it would work